scripts/azure/destroy-old-clusters.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
Why not in go?

Reply:
cloudtest use bash commands to create,destroy and control azure cloud. So bash scripts is the simpliest way to find and destroy old clusters

Reply:
These scripts will create more mainatners burden than htey deserve, please consider using `go`
I don't think that code like this line is something we should accept to maintain in 2020 and onwards:
```
 IFS=$'\t'; read -r -a cols <<< "${raws[$i-1]}"; unset IFS;
```

Reply:
We already have a bunch of scripts in bash, I don't think makes sense to rewrite all of them into go. 
Maybe we can rewrite that particular line in a more readable way? @nickolaev will it be ok to you?

Reply:
Having simple scripts may be OK, but these are too complex jsut to parse some arguments, You can parse the arguments in the go code itself. Why need these shell script magics.
A simple wrapper is OK, putting advanced logic is a potential source of problems in the future. All these clouds will change the way they operate in 6 months probably and then we (or whoever works on the project) will ave to figure out again how this code works. Go is a cleaner solution for all the advanced processing - arguments, algorithms, retries etc. Simple wrappers in shell are OK - similar to what #1973 does.

Reply:
That PR has quite simple logic, but it's written in complicated way. I believe we can review and simplify it, @alex-yust has thoughts about simplification

Reply:
@nickolaev I have simplified bash scripts. Could you please take a look at them.